packages/orbit-components/src/Box/index.js.flow

```diff
@@ -0,0 +1,147 @@
+// @flow
+/*
+  DOCUMENTATION: https://orbit.kiwi/components/Box/
+*/
+import * as React from "react";
+
+import type { Globals } from "../common/common.js.flow";
+
+// TODO: extend to suppressed and flat
+export type Elevation = "action" | "fixed" | "raised" | "overlay";
+
+export type ColorTokens =
+  | "productLight"
+  | "productLightHover"
+  | "productLightActive"
+  | "productNormal"
+  | "productNormalHover"
+  | "productNormalActive"
+  | "productDark"
+  | "productDarkHover"
+  | "productDarkActive"
+  | "productDarker"
+  | "white"
+  | "whiteHover"
+  | "whiteActive"
+  | "cloudLight"
+  | "cloudLightHover"
+  | "cloudLightActive"
+  | "cloudNormal"
+  | "cloudNormalHover"
+  | "cloudNormalActive"
+  | "cloudDark"
+  | "inkLighter"
+  | "inkLighterHover"
+  | "inkLighterActive"
+  | "inkLight"
+  | "inkLightHover"
+  | "inkLightActive"
+  | "inkNormal"
+  | "inkNormalHover"
+  | "inkNormalActive"
+  | "orangeLight"
+  | "orangeLightHover"
+  | "orangeLightActive"
+  | "orangeNormal"
+  | "orangeNormalHover"
+  | "orangeNormalActive"
+  | "orangeDark"
+  | "orangeDarkHover"
+  | "orangeDarkActive"
+  | "orangeDarker"
+  | "redLight"
+  | "redLightHover"
+  | "redLightActive"
+  | "redNormal"
+  | "redNormalHover"
+  | "redNormalActive"
+  | "redDark"
+  | "redDarkHover"
+  | "redDarkActive"
+  | "redDarker"
+  | "greenLight"
+  | "greenLightHover"
+  | "greenLightActive"
+  | "greenNormal"
+  | "greenNormalHover"
+  | "greenNormalActive"
+  | "greenDark"
+  | "greenDarkHover"
+  | "greenDarkActive"
+  | "greenDarker"
+  | "blueLight"
+  | "blueLightHover"
+  | "blueLightActive"
+  | "blueNormal"
+  | "blueNormalHover"
+  | "blueNormalActive"
+  | "blueDark"
+  | "blueDarkHover"
+  | "blueDarkActive"
+  | "blueDarker"
+  | "socialFacebook"
+  | "socialFacebookHover"
+  | "socialFacebookActive";
```

Comment:
I wish there was a way to not have to repeat all of this twice manually 😅 I'll think about automating this one day. (Just thinking out loud, feel free to resolve this comment.)